src/clr-core/button/button.element.ts

```diff
@@ -4,20 +4,55 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { baseStyles, CdsBaseButton, getElementWidth, property, registerElementSafely } from '@clr/core/internal';
+import {
+  badgeSlot,
+  baseStyles,
+  CdsBaseButton,
+  getElementWidth,
+  iconSpinner,
+  iconSpinnerCheck,
+  iconSlot,
+  property,
+  querySlot,
+  registerElementSafely,
+} from '@clr/core/internal';
+import '@clr/core/icon';
+import { ClarityIcons, errorStandardIcon } from '@clr/core/icon-shapes';
 import { html, query } from 'lit-element';
+import { styles as baseButtonStyles } from './base-button.element.css.js';
 import { styles } from './button.element.css.js';
 
+ClarityIcons.addIcons(errorStandardIcon);
+
+// TODO: when we migrate away from the base64 icons in other states, we will need to either move the consts
+// into this file or move the base button here. pulling cds-icons into core/internal creates a circular dependency
+const iconSpinnerError = html`<span class="button-status-icon"><cds-icon shape="error-standard"></cds-icon></span>`;
```

Comment:
Actually I can't. I initially had it in there but it uses `cds-icon` and importing `cds-icon` into base button caused a circular dependency because base button is delivered through `@clr/core/internal`.

Reply:
Leaving it as is until we resolve #4501